src/userlist/__tests__/userListReducers-test.js

```diff
@@ -0,0 +1,215 @@
+import { fromJS } from 'immutable';
+import { GET_USER_RESPONSE, PRESENCE_RESPONSE } from '../userListActions';
+import userListReducers, { activityFromPresence, timestampFromPresence } from '../userListReducers';
+
+it('when unrecognized action, no previous state, returns initial state, does not throw', () => {
```

Comment:
These test descriptions are a little hard to parse -- maybe we could rephrase them? Such as `it('returns initial state and does not throw for unknown action with no previous state')`..

I find it a little easier to read when the result comes first, followed by the conditions, but that may just be a personal preference